auditbeat/magefile.go

```diff
@@ -37,6 +37,8 @@ import (
 	// mage:import
 	"github.com/elastic/beats/v7/dev-tools/mage/target/integtest"
 	// mage:import
+	_ "github.com/elastic/beats/v7/dev-tools/mage/target/integtest/docker"
+	// mage:import
```

Comment:
Mage now [recommends](https://magefile.org/importing/) to follow the Go convention of no space between `//` and `mage` for directive comments.

Reply:
I changed this in all the magefiles I touched in this PR